### ISSUE TITLE
Search: English alt attribute for graphics is read in French accent in the French page

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#124]
URL: https://hcdev.openplus.ca/fr/avis-rappel/avis-rappel-aliments-allergene-rappel-certains-produits-vegetaliens-et-vegetariens
User Agent: Chrome, JAWS, Firefox, NVDA
Language: French

Issue Description: The graphics has English alt attribute in this French page and the English text is read in French accent.
![image](https://user-images.githubusercontent.com/88776404/137192361-ad048940-746d-4ab6-b9d7-2b4b77bb2df9.png)

Steps to reproduce:

1. Navigate to https://hcdev.openplus.ca/en/search/site
2. Enter "Veggie" to the Search field and activate the "Search" button
3. Switch to French page
4. Select the second item
5. Turn on screen readers
6. English alt attribute for the graphics is read in French accent in French page
Expected results: English text in French page should still be read in English. Add lang="en" to the <img> tags should resolve the issue.

**Comment by davidgotlieb** [comment]
Thanks @XiuJuanXia This would be a data entry issue. This was just a test being done by one of the programs so they may not have dealt with the alt tags as they would in a real situation. For example, this is what they used on the current site:

Happy Veggie World - « Patty de poulet croustillant (viande simulée) » - 3 kilogrammes

**Comment by echouin** [comment]
Marked as resolved as this is a user-generated issue.

**Comment by XiuJuanXia** [comment]
@echouin, I am curious that would users know they should upload French images with French alt attribute on the French page?

**Comment by echouin** [comment]
Hello @XiuJuanXia When uploading an image to the CMS website, the images will most likely be unilingual (or whatever they are using). However, they will be unable to upload the image w/o inputting alt text.

**Comment by XiuJuanXia** [comment]
@echouin So they don't have ability to enter the alt text?

**Comment by echouin** [comment]
@XiuJuanXia no, they have the ability to enter alt text in EN. Then when they switch over to FR, they would have the ability to enter the alt tags. I will check w/ OP and get back to you.

**Comment by echouin** [comment]
@XiuJuanXia From OP: this is Drupal behaviour, by default, you can't specify a language on an image upload field. I would imagine it's pretty edge case as well. In this instance it's english for testing. I would imagine that the alt text would always be in French, since alt text's job is to describe the image.

**Comment by mmiron-HC** [comment]
This is a non issue. If the users are responsible to upload images in both languages and enter alternative text to accompany the images, we will close this.
